packages/proto-signing/src/directsecp256k1hdwallet.ts

```diff
@@ -243,7 +243,7 @@ export class DirectSecp256k1HdWallet implements OfflineDirectSigner {
   /** Derivation instructions */
   private readonly accounts: readonly Secp256k1Derivation[];
 
-  protected constructor(mnemonic: EnglishMnemonic, options: DirectSecp256k1HdWalletConstructorOptions) {
+  constructor(mnemonic: EnglishMnemonic, options: DirectSecp256k1HdWalletConstructorOptions) {
```

Comment:
Why this (unrelated?) change?

Reply:
I want to manually construct an instance, not be forced to go via `fromMnemonic`. Going via `fromMnemonic` calls your custom BIP39 implementation.

Tbh this is not totally necessary with these changes, but I have a ts-ignore allowing it on my end for now.